lib/spack/docs/configuration.rst

```diff
@@ -688,10 +732,9 @@ down the source of the configuration:
    /home/myuser/spack/etc/spack/defaults/config.yaml:28    directory_layout: {architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}
    /home/myuser/spack/etc/spack/defaults/config.yaml:49    build_stage:
    /home/myuser/spack/etc/spack/defaults/config.yaml:50    - $tempdir/$user/spack-stage
-   /home/myuser/spack/etc/spack/defaults/config.yaml:51    - ~/.spack/stage
-   /home/myuser/spack/etc/spack/defaults/config.yaml:52    - $spack/var/spack/stage
-   /home/myuser/spack/etc/spack/defaults/config.yaml:57    source_cache: $spack/var/spack/cache
-   /home/myuser/spack/etc/spack/defaults/config.yaml:62    misc_cache: ~/.spack/cache
+   /home/myuser/spack/etc/spack/defaults/config.yaml:51    - $xdg_cache_home/stage
```

Comment:
(also noted elsewhere but it's now spack_cache_home - there are no longer any xdg_ config vars that the user can put in their config yaml)

Reply:
Done